proxy/examples/kafka/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:6.2.0
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-kafka:6.2.0
```

Comment:
Any reason for not using newest `7.0.1`?

Reply:
No reason. I copied the config from the confluent site, I'll bump the version to `7.0.1`